python/paddle/hapi/hub.py

```diff
@@ -329,3 +333,103 @@ def load(
     entry = _load_entry_from_hubconf(hub_module, model)
 
     return entry(**kwargs)
+
+
+def load_state_dict_from_url(
```

Comment:
Because we recommend users to use `Paddle.hub.load_state_dict_from_url` in [rfc](https://github.com/PaddlePaddle/community/blob/master/rfcs/APIs/20240916_api_design_for_load_state_dict_from_url.md?plain=1#L149), so we need to add it in \_\_all\_\_ list of `Paddle/python/paddle/hub.py`

Reply:
好的已经修改了